R/url_path.R

```diff
@@ -0,0 +1,20 @@
+#' Construct URL Path
+#' 
+#' Construct a URL from path components like \code{file.path}
+#'
+#' @param x a URL, or vector of URLs
+#' @param ... character vectors
+#' 
+#' @examples
+#' 
+#' url_path("https://github.com", c('Ironholds', 'nfultz'), 'urltools')
+#' 
+#' @export
+#'
+url_path <- function(x, ...) {
+  p <- path(x)
+  p[is.na(p)] = '.'
```

Comment:
The NA issue probably isn't going to be fixed; the others absolutely will be. In what situation does the NA issue block functionality like this?